api/graphql/makeModels.js

```diff
@@ -198,10 +201,7 @@ export default function makeModels (userId, isAdmin) {
             sort: sortBy,
             topic
           })
-      },
-      filter: nonAdminFilter(relation => relation.query(q => {
```

Comment:
Will fix. @robbiecarlton started off on the ticket and it was his fault. It wasn't me :)